src/applications/disability-calculator/tests/utils/helpers.unit.spec.js

```diff
@@ -1,211 +1,25 @@
-  },
-  {
-    rating: 0,
-    description: 'leg',
-  },
-  {
-    rating: 0,
-    description: 'ears',
-  },
-];
-
-describe('Disability Caclulator helper', () => {
-  it('should return an array with length of two elements', () => {
-    const result = calculateRating(ratingArray);
-    expect(result)
-      .to.be.an('array')
-      .that.has.length(2);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(noRatings)[0];
-    expect(result).to.be.equal(0);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArray)[0];
-    const actual = calculateRating(ratingArray)[1];
-    expect(result).to.be.equal(50);
-    expect(actual).to.be.equal(49.6);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArrayTwo)[0];
-    const actual = calculateRating(ratingArrayTwo)[1];
-    expect(result).to.be.equal(80);
-    expect(actual).to.be.equal(82.72);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArrayThree)[0];
-    const actual = calculateRating(ratingArrayThree)[1];
-    expect(result).to.be.equal(100);
-    expect(actual).to.be.equal(95.68);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArrayTens)[0];
-    const actual = calculateRating(ratingArrayTens)[1];
-    expect(result).to.be.equal(70);
-    expect(actual).to.be.equal(65.13);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArrayTwenties)[0];
-    const actual = calculateRating(ratingArrayTwenties)[1];
-    expect(result).to.be.equal(40);
-    expect(actual).to.be.equal(36);
-  });
+/**
+ * The expected result of these tests came from manually determining
+ * the combined rating using the data tables located at
+ * https://www.benefits.va.gov/compensation/rates-index.asp.
+ */
 
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArraySix)[0];
-    const actual = calculateRating(ratingArraySix)[1];
-    expect(result).to.be.equal(90);
-    expect(actual).to.be.equal(89.2);
-  });
-
-  it('should calculate ', () => {
-    const result = calculateRating(ratingArraySeven)[0];
-    const actual = calculateRating(ratingArraySeven)[1];
-    expect(result).to.be.equal(100);
-    expect(actual).to.be.equal(99);
+import { expect } from 'chai';
+import inputs from './inputs.json';
+
+import { calculateCombinedRating } from '../../utils/helpers';
+
+describe('Disability Caclulator helpers', () => {
+  describe('calculateCombinedRating', () => {
+    for (const input of inputs) {
+      it('should calculate the correct result', () => {
+        const { ratings, combinedRating: expectedResult } = input;
+        const result = calculateCombinedRating(ratings);
+
+        expect(result).to.have.keys(['exact', 'rounded']);
+        expect(result.exact).to.be.equal(expectedResult.exact);
+        expect(result.rounded).to.be.equal(expectedResult.rounded);
+      });
+    }
```

Comment:
Is it bad practice to do an `it()` inside of a loop?

Reply:
I don't think so, but you might want to make the name of the test ("should calculate the correct result") unique to each iteration, so that you can tell which one fails.

Reply:
That is an awesome suggestion - thank you!